lib/config/definitions.js

```diff
@@ -980,6 +980,19 @@ const options = [
     mergeable: true,
     cli: false,
   },
+  {
+    name: 'nuget',
+    description: 'Configuration object for C#/Nuget',
+    stage: 'package',
+    type: 'json',
+    default: {
+      enabled: true,
+      fileMatch: ['(^|/)*\\.csproj$'],
+      includePreRelease: false,
```

Comment:
We have already been using the term "unstable" in JS which covers alpha, beta, pre-release, etc. Would it be possible to reuse the same terminology for nuget and save this extra term?

Reply:
We could do it that way yes, although the term pre-release is common in the .net world (https://docs.microsoft.com/en-us/nuget/create-packages/prerelease-packages). But we can probably sort it out via documentation.

Reply:
The term is `ignoreUnstable` and it defaults to `false` so you won't need to explicitly configure it in this block